.vscode/settings.json

```diff
@@ -31,5 +31,73 @@
     "source.organizeImports": true
   },
   "eslint.format.enable": true,
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "files.associations": {
+    "any": "cpp",
```

Comment:
Hmmm was this edit intended? .any isn't usually a C++ file

Reply:
Yes, codespaces keeps making these changes automatically and I can't figure out how to suppress it :(